client/cmd/utils/flags.go

```diff
@@ -147,39 +146,25 @@ var (
 		Name:  "fast",
 		Usage: "Enable fast syncing through state downloads (replaced by --syncmode)",
 	}
-	LightModeFlag = cli.BoolFlag{
-		Name:  "light",
-		Usage: "Enable light client mode (replaced by --syncmode)",
-	}
 	defaultSyncMode = knode.DefaultConfig.SyncMode
 	SyncModeFlag    = TextMarshalerFlag{
 		Name:  "syncmode",
-		Usage: `Blockchain sync mode ("fast", "full", or "light")`,
+		Usage: `Blockchain sync mode ("fast" or "full")`,
```

Comment:
Why do we remove light flag and mode?

Reply:
because we do not support it atm

Reply:
May be it'd be better to remove flags in a new issue. It's a bit confusing to see it in gas issue.